DeskKit/DeskKit/View Models/DKContactUsViewModel.m

```diff
@@ -291,9 +291,11 @@ - (NSDictionary *)caseDictionary
 - (NSDictionary *)messageDictionary
 {
     NSString *fromEmail = [self bestFromEmail];
+    NSString *body = self.bodyItem.text.string.dkTrimmedString;
+    
     NSMutableDictionary *dictionary = [@{
                                          DKMessageDirectionKey: @"in",
-                                         DKMessageBodyKey: self.bodyItem.text.string,
```

Comment:
I don't see a reason to not to trim. If we don't trim it on client side, API will trim it anyway. That's the behavior I notice.